addons/web/static/src/views/relational_model/static_list.js

```diff
@@ -875,7 +875,7 @@ export class StaticList extends DataPoint {
         await Promise.all(proms);
 
         await this._sort();
-        this._onUpdate();
+        await this._onUpdate();
```

Comment:
qunit for this? resequence, block onchange on main record, save -> should not wait for onchange to return to save